components/Setup/Main.vue

```diff
@@ -135,7 +135,7 @@ npm install</pre
               <p class="c-content-main__more">
                 Once you’ve completed building PacketCrypt, you can begin
                 mining.
-                <nuxt-link to="/">Learn More</nuxt-link>
+                <nuxt-link to="https://docs.pkt.cash/en/latest/">Learn More</nuxt-link>
```

Comment:
I think these `to=` links cause nuxt/vue it to create only internal links and you need to use href to get a proper external link.